### ISSUE TITLE
recorder.js

### ISSUE THREAD

**Issue by llaine** [pfernandez/songnotes#4]
Hi, 

i'm trying to use your project, and i wonder how you load recorder.js library.
Whenever i try to use it, this doesn't work and told me that the recorderWorker.js is 'Resource interpreted as Script but transferred with MIME type text/html'. 

I'm kind of stuck and i wonder how you manage this.

ty

**Comment by pfernandez** [comment]
I don't have time to look into this at the moment, but the first thing I'd check is what version of Meteor you're using. The last version I tried it with was 0.8.2, so it's pretty far behind at the moment. Try running Meteor with `meteor --release 0.8.2` and see if that helps.

**Comment by llaine** [comment]
Ok, apparently this doesn't look like a meteor problem.
Any issues when you import the recorder.js file the first time and then use it? 

It's so frustrating

**Comment by pfernandez** [comment]
No, I haven't had any issues with Recorder.js. The warning "Resource interpreted as Script but transferred with MIME type text/html" isn't really an error that should break things, although I haven't been seeing it myself so something is definitely amiss.

I'll try to get in there this week or next, update everything to work with the latest version of Meteor, and post some instructions on how to run Songnotes. But be warned that it's in development and is definitely still buggy.
